src/operations/retakeExams/RetakeExamList.tsx

```diff
@@ -26,42 +26,43 @@ const RETAKE_EXAM_LIST_SX = {
   },
 };
 
+// TODO: add session list before rattrapage list for specific students
 export const RetakeExamList = () => {
   const studentId = authProvider.getCachedWhoami()?.id;
-  const {data: session, isLoading} = useGetList("retakeExams-sessions");
-  const sessionId = session?.[0]?.id;
+  const {data: sessions, isLoading} = useGetList("retakeExams-sessions");
+  const sessionId = sessions?.[0]?.id;
```

Comment:
```suggestion
  const {data: sessions = [], isLoading} = useGetList("retakeExams-sessions");
```

below the session list loading

add an extra statement 

```ts
if (!sessions.length) {
  // return an error  card or something that says there is no available session catch-up sessions
}

// NPEs's been correctly checked beforehand
const sessionId = sessions[0].id;
```


only then will you return the list of catch-up•s